indico/modules/rb/client/js/common/bookings/BookingDetails.jsx

```diff
@@ -75,6 +75,9 @@ class BookingDetails extends React.Component {
     occurrencesVisible: false,
     activeConfirmation: null,
     acceptanceFormVisible: false,
+    preBookingWarningVisible: false,
+    concurrentBookings: [],
+    acceptanceData: null,
```

Comment:
I'd merge those into a single state field, since you usually change all of them at the same time anyway.